catalog.json

```diff
@@ -1,20 +1,21 @@
 {
   "id": "8478b533-2b59-4007-8494-2feec5970f94",
   "name": "cdn-route",
-  "description": "CDN Route Service",
+  "description": "Custom URL, CDN, and free TLS certificates",
   "bindable": true,
   "metadata": {
-    "displayName": "cdn-route"
+    "displayName": "Content Distribution Network",
+    "documentationUrl": "https://cloud.gov/docs/services/cdn-route"
```

Comment:
Suggest including the / at the end of the URL (https://cloud.gov/docs/services/cdn-route/) for consistency.